docs(dns): add docs for dns&domain product (#909)
scaleway/docs-content: network/dns-cloud/how-to/unlock-internal-domain.mdx

---
meta:
  title: How to unlock an internal domain name

>   description: This page explains how to unlock an internal domain name with Scaleway Domains & DNS.

Reviewer: ```suggestion
  description: This page explains how to unlock an internal domain name with Scaleway Domains and DNS.
```

content:
  h1: How to unlock an internal domain name

>   paragraph: This page explains how to unlock an internal domain name with Scaleway Domains & DNS.

Reviewer: ```suggestion
  paragraph: This page explains how to unlock an internal domain name with Scaleway Domains and DNS.
```

dates:
  validation: 2022-10-12
  posted: 2022-10-12
---

Domain names are locked by default to avoid any unsolicited outgoing transfer. In case you want to transfer your domain name to another registrar, you can unlock it from your Scaleway console. 

<Message type="requirement">

- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 

> - You have a domain name registered with Scaleway Domains & DNS

Reviewer: ```suggestion
- You have a domain name registered with Scaleway Domains and DNS
```

</Message>

## How to unlock a domain name

> 1. Click **Domains & DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 

Reviewer: ```suggestion
1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 
```

>  The Domain & DNS dashboard displays. 

Reviewer: ```suggestion
 The Domains and DNS dashboard displays. 
```

 
2. Click the domain name you want to configure.

> 3. Click **Unlock** next to **Transfer** in the domain information section. A pop-up displays.

Reviewer: ```suggestion
3. Click **Unlock** under **Transfer** in the domain information section. A pop-up displays.
```

    <Lightbox src="scaleway-domain-unlock.webp" alt="" />

> 4. Click **Validate** to unlock your domain name.

Reviewer: ```suggestion
4. Click **Unlock** to unlock your domain name.
```

> 5. Click <Icon name="copy" /> to copy the authentication code for your domain name and paste it in the form at your new registrar. 

Reviewer: This step doesn't exist anymore, we can remove it and the message box.

    <Lightbox src="scaleway-domain-copy-auth.webp" alt="" />

    <Message type="important">

    * The authentication key is a secret key that launches the transfer of your domain name. We recommend you take note of it and not share it with anyone.
    * Make sure your domain name is locked at all times, unless you want to transfer it. 

    </Message>

## How to lock a domain name

> 1. Click **Domains & DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domain & DNS dashboard displays. 

Reviewer: ```suggestion
1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Domains and DNS dashboard displays. 
```

2. Click the domain name you want to configure.

> 3. Click **Lock** next to **Transfer** in the domain information section.

Reviewer: ```suggestion
3. Click **Lock** under **Transfer** in the domain information section. A pop-up displays asking you to confirm.
```

> 

Reviewer: ```suggestion
4. Click **Lock** to confirm. Your domain will be locked in a matter of minutes.
```
